src/Xenial.Tasty/Plugins/PluginLoader.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xenial.Delicious.Scopes;
+
+namespace Xenial.Delicious.Plugins
+{
+    public class PluginLoader
+    {
+        public Task LoadPlugins(TastyScope scope)
+        {
+            var entryAssembly = Assembly.GetEntryAssembly();
+
+            var pluginAttributes = entryAssembly.GetCustomAttributes(typeof(TastyPluginAttribute), true).OfType<TastyPluginAttribute>();
+
+            foreach (var pluginAttribute in pluginAttributes)
+            {
+#if DEBUG
```

Comment:
We should remove them, but for now I guess they are valuable until architecture is stable